subt/ros/robot/launch/freyja.launch

```diff
@@ -200,21 +138,51 @@
                                    true,  true,  true,
                                    false, false, false,
                                    true,  true,  true,
-                                   true,  true,  true] </rosparam>
+                                   true,  true,  true]</rosparam>
 
-    <!-- The unit of the threshold is a Mahalanobis distance (i.e. sum) of numbers of sigmas in each dimension -->
+    <!-- When the robot starts slipping or spinning wheels in the air, we need to start ignoring wheel-based odometry.
+         The unit of the threshold is a Mahalanobis distance (i.e. sum) of numbers of sigmas in each dimension -->
     <rosparam param="odom0_twist_rejection_threshold">4</rosparam>
 
+    <!-- Trying to not have data stuck in a TCP buffer. -->
+    <rosparam param="odom0_nodelay">true</rosparam>
+    <rosparam param="odom1_nodelay">true</rosparam>
+    <rosparam param="pose0_nodelay">true</rosparam>
+    <rosparam param="pose1_nodelay">true</rosparam>
+    <rosparam param="imu0_nodelay">true</rosparam>
+
+    <!-- Setting some physical constraints. -->
+    <rosparam param="acceleration_limits">[10.6, 0.0, 0.0,
+                                           0.0, 0.0, 1.6]</rosparam>
+    <rosparam param="deceleration_limits">[10.6, 0.0, 0.0,
+                                           0.0, 0.0, 1.6]</rosparam>
+
     <!-- When some data arrive late, we are willing to go back in time and correct the estimate. -->
     <param name="smooth_lagged_data" value="true"/>
     <param name="history_length" value="1.0"/>
 
-    <!-- -->
+    <!--
     <param name="print_diagnostics" value="true"/>
     <remap from="/diagnostics" to="$(arg robot_name)/odom_diagnostics"/>
-    <!-- -->
+    -->
 
+    <!-- Output -->
     <remap from="odometry/filtered" to="/$(arg robot_name)/odom_fused"/>
   </node>
 
+  <!-- Merging both RGBD cameras into a single stream. -->
+  <node pkg="topic_tools" type="relay" name="points_merge_front" args="$(arg robot_name)/rgbd_front/points $(arg robot_name)/rgbd_all/points"/>
+  <node pkg="topic_tools" type="relay" name="points_merge_rear" args="$(arg robot_name)/rgbd_rear/points $(arg robot_name)/rgbd_all/points"/>
+
+  <node ns="mapping" pkg="octomap_server" type="octomap_server_node" name="octomap">
+    <remap from="cloud_in" to="/$(arg robot_name)/rgbd_all/points"/>
+    <param name="frame_id" value="odom"/>
+    <!-- In case we want to filter out ceilings.
+    <param name="pointcloud_max_z" value="1.5"/>
+    -->
+    <!-- We trust the simulated sensor a lot more than what default parameters expect. -->
+    <param name="sensor_model/hit" value="0.95"/>
+    <param name="sensor_model/miss" value="0.05"/>
+  </node>
```

Comment:
As I noted in email, adding this here at this time is not a good idea since we are already and full cpu load due to opencv dnn cpu implementation.